dhis-2/dhis-api/src/main/java/org/hisp/dhis/reservedvalue/ReservedValueStore.java

```diff
@@ -38,13 +38,15 @@
     extends GenericStore<ReservedValue>
 {
 
+    void bulkInsertReservedValues( List<ReservedValue> toAdd );
+
     void reserveValues( List<ReservedValue> toAdd );
 
-    List<ReservedValue> reserveValuesAndCheckUniqueness( ReservedValue reservedValue, List<String> values );
+    List<ReservedValue> getAvailableValues( ReservedValue reservedValue, List<String> values );
 
-    List<ReservedValue> reserveValuesJpa( ReservedValue reservedValue, List<String> values );
+    List<ReservedValue> getReservedValues( ReservedValue reservedValue, List<String> values );
 
-    List<ReservedValue> getIfReservedValues( ReservedValue reservedValue, List<String> values );
+    List<ReservedValue> reserveValuesJpa( ReservedValue reservedValue, List<String> values );
```

Comment:
method name in an interface should not contain reference on its actual implementation.
I would remove "Jpa" part from the method unless there's a very good reason to keep it.

Reply:
That is an old method, probably used in older versions , i didn't delete as i didn't want to have troubles with older versions but we can rename with a more sensible name